Jenkinsfile

```diff
@@ -62,7 +62,7 @@ pipeline {
                 branch 'master'
             }
             steps {
-                sh './mvnw deploy -Papache-release -Dgpg.skip=true -DskipTests --batch-mode -pl -:brave-itests -nsu'
+                sh './mvnw deploy:deploy -Papache-release -Dgpg.skip=true -DskipTests --batch-mode -pl -:brave-itests -nsu'
```

Comment:
`-DskipTests` becomes a bit superfluous since the test phase shouldn't be run anyways.